src/command/expire/expire.c

```diff
@@ -727,6 +727,37 @@ removeExpiredArchive(InfoBackup *infoBackup, bool timeBasedFullRetention)
                             // Log if there is more to log
                             else
                                 logExpire(&archiveExpire, archiveId);
+
+                            // Based on the timeline of the backupArchiveStart, look for history files to expire
+                            StringList *historyFilesList =
+                                strLstSort(
+                                    storageListP(
+                                        storageRepo(),
+                                        strNewFmt(STORAGE_REPO_ARCHIVE "/%s", strZ(archiveId)),
+                                        .expression = STRDEF("^[0-F]{8}.history$")),
```

Comment:
This seems like a useful constant that should be added to `src/command/archive/common.h`.

Reply:
Good point. Moved it (for the next commit)